tests/unit/test_locations_feature_engineering.py

```diff
@@ -53,6 +53,14 @@ def test_main_explodes_region_columns(self):
 
         self.assertEqual(rows[6].yorkshire_and_the_humbler, 1)
 
+    def test_main_adds_date_diff_column(self):
+        df = locations_feature_engineering.main(self.PREPARED_LOCATIONS_TEST_DATA)
+        self.assertIn("date_diff", df.columns)
+
+        rows = df.collect()
+
+        self.assertEqual(rows[7].date_diff, 52)
+
```

Comment:
Can we add one more for the actual max date? 
```suggestion
self.assertEqual(rows[13].date_diff, 0)
```